docs/api/sql/Raster-operators.md

```diff
@@ -18,6 +18,52 @@ Output:
 512
 ```
 
+### RS_ScaleX
+
+Introduction: Returns the pixel width of the raster in map units.
```

Comment:
Call this `CRS unit` (Coordinate Reference System)

Reply:
done